lib/hawkular/metrics/metric_api.rb

```diff
@@ -34,9 +34,10 @@ def push_data(gauges: [], counters: [], availabilities: [])
       gauges.each { |g| default_timestamp g[:data] }
       counters.each { |g| default_timestamp g[:data] }
       availabilities.each { |g| default_timestamp g[:data] }
-
       data = { gauges: gauges, counters: counters, availabilities: availabilities }
-      http_post('/metrics/data', data)
+      path = 'metrics/'
```

Comment:
before this PR, the call was to `http_post('/metrics/data', data)` where the path (metrics) begins with a `/`
after the change, it looks like path doesn't start with the slash `/`
is this possibly a regression here?